conference/CG/sig.yml

```diff
@@ -1,13 +1,13 @@
 - title: ACM SIGGRAPH
-  year: 2021
-  id: sig21
-  description: THE 48TH INTERNATIONAL CONFERENCE & EXHIBITION ON COMPUTER GRAPHICS & INTERACTIVE TECHNIQUES
-  link: https://s2021.siggraph.org/
-  deadline: '2021-01-28 22:00:00'
-  abstract_deadline: '2021-01-27 22:00:00'
+  year: 2022
+  id: sig22
+  description: THE 49TH INTERNATIONAL CONFERENCE & EXHIBITION ON COMPUTER GRAPHICS & INTERACTIVE TECHNIQUES
+  link: https://s2022.siggraph.org/
+  deadline: '2021-01-27 22:00:00'
```

Comment:
You need to keep historical records and add the next data at the end instead of modifying it directly.